### PR TITLE
Replacing link text URLs with text

### LINKED ISSUE #647: iPhone display error
On the iPhone we have reports that some anchor elements on that page that aren't wrapping, causing the page to be ready wide on mobile. She is able to horizontally scroll or zoom out on the page, but it doesn't look crisp like it does on an Android. 

Specific page causing grief is:
https://accessibility.civicactions.com/posts/automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe

### PR DESCRIPTION
**Pull request by dmundra**
Resolves #647 (albeit not permanently).

### CODE REVIEW

**Comment by dmundra** [comment]
@mgifford a more permanent fix would be to consider either JS or CSS from articles:
* https://css-tricks.com/better-line-breaks-for-long-urls/
* https://css-tricks.com/where-lines-break-is-complicated-heres-all-the-related-css-and-html/

I think for now, fixing the post is good enough. What do you think?

File: _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md [hunk 1]
```diff
@@ -195,9 +195,9 @@ As noted in our [playbook for automated testing](/playbook/automated-testing):
...
 * Examples of applying the above steps to Jekyll sites:
-  * [BBC's "Accessibility, News and You"](https://github.com/bbc/accessibility-news-and-you) can be seen in this pull request: [https://github.com/dmundra/accessibility-news-and-you/pull/1](https://github.com/dmundra/accessibility-news-and-you/pull/1).
-  * [GSA's "Accessibility for Teams"](https://github.com/GSA/accessibility-for-teams) can be seen in this pull request: [https://github.com/dmundra/accessibility-for-teams/pull/1](https://github.com/dmundra/accessibility-for-teams/pull/1).
-  * [18F Accessibility Guide](https://github.com/18F/accessibility) can be seen in this pull request: [https://github.com/dmundra/accessibility-1/pull/1](https://github.com/dmundra/accessibility-1/pull/1).
+  * [BBC's "Accessibility, News and You"](https://github.com/bbc/accessibility-news-and-you) can be seen in [this pull request](https://github.com/dmundra/accessibility-news-and-you/pull/1).
+  * [GSA's "Accessibility for Teams"](https://github.com/GSA/accessibility-for-teams) can be seen in [this pull request](https://github.com/dmundra/accessibility-for-teams/pull/1).
```
Comment (mgifford) [inline _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md:199]: Ok, but lets make the link more descriptive. We shouldn't have 2 links called "pull request" going to different places. Imagine if we're listing this by links without the other context.

File: _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md [hunk 2]
```diff
@@ -99,9 +99,9 @@ Running Pa11y on 49 URLs:
 ✔ 49/49 URLs passed
 ```
 
-As you can see, at the time of this writing we have all our pages passing! To see examples of us fixing reported accessibility issues check out [https://github.com/CivicActions/accessibility/pull/128](https://github.com/CivicActions/accessibility/pull/128).
+As you can see, at the time of this writing we have all our pages passing! To see examples of us fixing reported accessibility issues check out [pull request #128](https://github.com/CivicActions/accessibility/pull/128).
```
Comment (mgifford) [inline _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md:102]: For consistency both this and the next link probably should be:
- pull request #128 on accessibility site
- pull request #128 on pa11y-ci repo

**Comment by mgifford** [comment]
For accessibility it is better not to have the URL, but to write descriptive text. So we can probably avoid that JS or CSS hack. But I do wonder if we should look for long URLs that we link to as part of our ci/cd process. We obviously don't want to eliminate all links, because sometimes we do want to talk about say CivicActions.ca specifically. However, we can look for long links like these.

File: _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md (hunk 2 above)
Comment (dmundra) [reply _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md:102]: The second one is an issue. How about this for link text:

* `pull request #128 on this site`
* `issue #128 on the pa11y-ci repository`

File: _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md (hunk 1 above)
Comment (dmundra) [reply _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md]: Updated. How does that look?

**Comment by dmundra** [comment]
> For accessibility it is better not to have the URL, but to write descriptive text. So we can probably avoid that JS or CSS hack. But I do wonder if we should look for long URLs that we link to as part of our ci/cd process. We obviously don't want to eliminate all links, because sometimes we do want to talk about say CivicActions.ca specifically. However, we can look for long links like these.

Long term if it is a problem we should consider either checking for this or just letting the user know to adjust the link text.

File: _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md (hunk 2 above)
Comment (mgifford) [reply _posts/2021-03-12-automated-accessibility-testing-leveraging-github-actions-and-pa11y-ci-with-axe.md]: That's just great.

**Review by mgifford — APPROVED** [review]
Great change. Happy to report that we fixed the iPhone error with this.

**Pull request merged**